src/lib/device-config.ts

```diff
@@ -41,10 +41,10 @@ export const generateConfig = (
 ) => {
 	const userPromise = getUser(req);
 
-	// Devices running ResinOS >= 2.0 can use Registry v2
-	const registryHost = resinSemver.satisfies(osVersion, '<2.0.0')
-		? REGISTRY_HOST
-		: REGISTRY2_HOST;
+	// Devices running ResinOS >= 1.2.1 can use Registry v2
+	const registryHost = resinSemver.satisfies(osVersion, '>=1.2.1')
```

Comment:
This returns false for `2.0.0-betaX` which would mean those versions would suddenly use `REGISTRY_HOST`, this should be
```suggestion
	const registryHost = resinSemver.satisfies(osVersion, '<1.2.1')
		? REGISTRY_HOST
		: REGISTRY2_HOST;
```
instead

Reply:
@Page- what the heck, I **did not** expect that, but checks out.

```
> let resinSemver = require('resin-semver');
undefined
> let osVersion = '2.0.0-beta1'
undefined
> resinSemver.satisfies(osVersion, '>=1.2.1')
false
```

 Will do the changes as you suggest. I thought it was clearer this original way, but it shouldn't matter much.

Reply:
Went with `.gte` for clarity.